pkg/clientcert/cert_controller.go

```diff
@@ -399,7 +400,7 @@ func jitter(percentage float64, maxFactor float64) float64 {
 	if maxFactor <= 0.0 {
 		maxFactor = 1.0
 	}
-	newPercentage := percentage + percentage*rand.Float64()*maxFactor
+	newPercentage := percentage + percentage*secureRandFloat64()*maxFactor
```

Comment:
I am not quite clear why we need a secure rand here, it is to produce a jitter time isn't it?

Reply:
Agree, let me set it to false positive alarm.